templates/base/packages/nextjs/utils/scaffold-eth/getMetadata.ts.template.mjs

```diff
@@ -14,7 +17,7 @@ export const getMetadata = ({
   description: string;
   imageRelativePath?: string;
 }): Metadata => {
-  const imageUrl = `${baseUrl}${imageRelativePath}`;
+  const imageUrl = \`\${baseUrl}\${imageRelativePath}\`;
 
   return {
```

Comment:
>  don't understand how to templatize getMetadata.ts

I think we can wrap this whole object inside the`${stringify({...})}` then inside you can `extraMetadata` ? 

![Screenshot 2024-11-04 at 9 14 08 PM](https://github.com/user-attachments/assets/2fe236ea-8beb-4d0e-9dc6-a8d819f52975)

Reply:
Forgive me if i get this wrong but then it cannot read baseUrl (If we stringify the return)
![image](https://github.com/user-attachments/assets/337b4331-f1b4-4d61-b30b-2f32f529a0d3)

Reply:
Ohh yeah makes sense, my bad. Can we have an corresponding args.mjs file for this file? 

I don't see it in https://github.com/portdeveloper/manifest-test

Reply:
`export const titleTemplate = "%s | My PWA App"

export const extraIcons = {
  shortcut: "/favicon.ico",
  apple: [{ url: "/icons/apple-touch-icon.png", sizes: "180x180" }]
}

export const extraMetadata = {
  applicationName: 'title',
  manifest: "/manifest.json",
  appleWebApp: {
    capable: true,
    statusBarStyle: "default",
    title: 'title'
  },
  formatDetection: {
    telephone: false
  }
}

`

Reply:
https://github.com/portdeveloper/test-everything Here is a repo with all the changed files.

Reply:
> Forgive me if i get this wrong but then it cannot read baseUrl (If we stringify the return)

If I understand it right you can theoretically do (writing it without trying)

```
return {
  metadataBase: new Url(baseUrl),
  ...${stringify({title: ..., description: ... , openGraph: .., twitter: ..}),
  etc
```

but since that part is always the same I think it's ok that it already stringifyed

Reply:
If we only partially stringify something, the passed args does not work
e.g. 
![image](https://github.com/user-attachments/assets/06078986-9330-4206-beef-608a5a13374f)